chart/compass/charts/operations-controller/templates/rbac.yaml

```diff
@@ -30,6 +30,18 @@ rules:
   - get
   - update
   - patch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
What is the purpose of these new leases resources and is this configuration auto-generated or manually added?

Reply:
Those `leases` are a Kubernetes concept for concurrency management. The standard generated RBAC settings from the new v3 Kubebuilder were containing this additional `leases` resource. Without it, the controller doesn't function correctly.